### PR TITLE
"Hello, World" in README to help newcomers

### PR DESCRIPTION
**Pull request by reinkrul**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -23,8 +25,28 @@ Distributed registry for storing and querying health care providers their vendor
...
+
+The simplest way to spin up the Nuts stack is by using the setup provided by `nuts-network-local <https://github.com/nuts-foundation/nuts-network-local>`_.
+The setup is meant for development purposes and starts a Nuts node, "Demo EHR", "Registry Admin Demo" for administering your vendor and care organizations and a HAPI server to exchange FHIR data.
+
+To get started, clone the repository and run the following commands to start the stack:
+
+.. code-block:: shell
+
+    cd single
+    docker-compose pull
```
Comment (woutslakhorst) [inline README.rst:39]: its `docker compose pull/up` these days. Minus the hyphen

File: README_template.rst [hunk 2]
```diff
@@ -23,17 +25,26 @@ Distributed registry for storing and querying health care providers their vendor
...
+
 .. include:: docs/pages/development/1-development.rst
     :start-after: .. marker-for-readme
 
 Configuration
-*************
+^^^^^^^^^^^^^
 
-.. include:: docs/pages/configuration/configuration.rst
+.. include:: docs/pages/getting-started/0-hello-world-docker.rst
```
Comment (woutslakhorst) [inline README_template.rst:43]: You made a booboo

**Review by woutslakhorst — CHANGES_REQUESTED** [review]
README is al borked

**Comment by reinkrul** [comment]
> README is al borked

?

File: docs/pages/getting-started/0-hello-world-docker.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.. _running-docker:
```
Comment (woutslakhorst) [inline docs/pages/getting-started/0-hello-world-docker.rst:1]: duplicate tag:

```
/Users/wout.slakhorst/nedap_healthcare/nuts-node/docs/pages/getting-started/0-hello-world-docker.rst:4: WARNING: duplicate label running-docker, other instance in /Users/wout.slakhorst/nedap_healthcare/nuts-node/docs/pages/getting-started/1-running-docker.rst
/Users/wout.slakhorst/nedap_healthcare/nuts-node/docs/pages/getting-started/1-running-docker.rst:4: WARNING: duplicate label running-docker, other instance in /Users/wout.slakhorst/nedap_healthcare/nuts-node/docs/pages/getting-started/0-hello-world-docker.rst
```

found using:
`cd docs & make html`

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

**Review by woutslakhorst — APPROVED** [review]

**Pull request merged**